packages/stylelint-plugin/src/rules/use-deprecated-component-library-helpers-scss-imports.ts

```diff
@@ -0,0 +1,85 @@
+import { Root, AtRule } from "postcss"
+import {
+  deprecatedComponentLibraryColorImport,
+  deprecatedComponentLibraryTypeImport,
+  deprecatedComponentLibraryLayoutImport,
+} from "../messages"
+import { Options, RuleDefinition } from "../types"
+
+type ImportToReplace = {
+  oldImport: string
+  newImport: string
+  message: any
+}
+
+const importsToReplace: ImportToReplace[] = [
+  {
+    oldImport: "~@kaizen/component-library/styles/type",
+    newImport: "~@kaizen/deprecated-component-library-helpers/styles/type",
+    message: deprecatedComponentLibraryTypeImport,
+  },
+  {
+    oldImport: "~@kaizen/component-library/styles/color",
+    newImport: "~@kaizen/deprecated-component-library-helpers/styles/color",
+    message: deprecatedComponentLibraryColorImport,
+  },
+  {
+    oldImport: "~@kaizen/component-library/styles/layout",
+    newImport: "~@kaizen/deprecated-component-library-helpers/styles/layout",
+    message: deprecatedComponentLibraryLayoutImport,
+  },
+]
+
+export const useDeprecatedComponentLibraryHelpersScssImports: RuleDefinition = {
```

Comment:
What this rule is and does should be described in the README.

Reply:
done